docs/pages/about/about_ro_crate.md

```diff
@@ -12,107 +12,111 @@ sidebar: about
-
-* researchers engaged with computation and data-intensive, workflow-driven analysis
-* digital repository managers and infrastructure providers
-* individual researchers looking for a straight-forward tool or how-to guide to “FAIRify” their data
-* data stewards supporting research projects in creating and curating datasets.
-
-RO-Crates are suitable for any domain, as they are general-purpose and can contain arbitrary data, but they can also be specialised through the use of [RO-Crate Profiles](profiles). Profiles define additional conventions, types and properties that RO-Crates may use within a particular domain, application or framework. For example, the [Workflow RO-Crate profile](https://w3id.org/workflowhub/workflow-ro-crate/) provides guidance on constructing RO-Crates that describe executable workflows.
-
-We continue to gather [use cases](use_cases), please help us by adding more.
-
-## About the RO-Crate Community
-
-The [RO-Crate specification](specification) is developed openly on GitHub by research community members across the globe. RO-Crate [tools](tools) and [profiles](profiles) are also community-developed and open-source. 
-
-Anyone can join the RO-Crate community and contribute to these outputs. See the [Community](community) pages for more information.
+An RO-Crate is an integrated view through which you can see an entire [Research Object](background#research-object-background);
+the methods, the data, the output and the outcomes of a project or a piece of work. 
+Linking all this together enables the sharing of research outputs with their context,
+as a coherent whole.
+
+RO-Crates link data and metadata no matter where they are stored --
+so that from a paper, you can find the data, and from the data, you can find its authors, and so on.
+
+For example, an RO Crate won't just contain an author's name.
+It would also contain their ORCID, which in turn is connected to their affiliations,
+their funding, and their other publications.
+
+<figure>
+  <img src="../assets/img/ro-crate_packaging.png" alt="Diagram representing RO-Crate connections. An open cardboard box labelled 'Describe package and parts' contains people with ORCIDs, and lab equipment with RRIDs. The box is linked through a DOI to a bubble labelled 'Integrated view', which contains other types of research object. Each type of object has an arrow pointing to an example repository or registry for that data, such as workflows to WorkflowHub, presentations to PubMed, and data to databases." />
+  <figcaption>RO-Crates can link and be linked to research objects from many different sources using persistent identifiers like ORCIDs and DOIs. <br/>Image credit: Goble, C. (2024, February 16). FAIR Digital Research Objects: Metadata Journeys. University of Auckland Seminar, Auckland. Zenodo. <a href="https://doi.org/10.5281/zenodo.10710142">https://doi.org/10.5281/zenodo.10710142</a> </figcaption>
```

Comment:
@douglowe happy with this alt text + caption?

Reply:
Looks good to me!